core/arch/arm/mm/tee_pager.c

```diff
@@ -1069,11 +1098,11 @@ static struct tee_pager_pmem *tee_pager_get_page(struct tee_pager_area *area)
 		uint32_t a;
 
 		assert(pmem->area && pmem->area->pgt);
+		pgt_dec_used_entries(pmem->area->pgt);
```

Comment:
That shouldn't matter, this is still supposed to be atomic for the other cores. I'm OK with changing though.